staging/staging-ec2/terragrunt.hcl

```diff
@@ -0,0 +1,24 @@
+locals {
+  env_variables = read_terragrunt_config(find_in_parent_folders("env.hcl"))
+  environment   = local.env_variables.locals.environment
+
+}
+
+terraform {
+  source = "github.com/flugel-it/workshop-1-terraform-module"
+}
+
+include {
+  path = find_in_parent_folders()
+}
+
+inputs = {
+    cidr_block       = "10.0.0.0/16"
+    instance_tenancy = "default"
+    vpc_id           = aws_vpc.main.id
```

Comment:
these variables are not expected in the ec2 module, remove these lines
you can check the expected variables [here](https://github.com/flugel-it/workshop-ec2/blob/main/variables.tf)